### PR TITLE
feat(ECC):Added pagination to ECC release list

### PR DESCRIPTION
**Pull request by eldrin30**
[//]: # (Copyright Bosch.IO GmbH 2020)
[//]: # (This program and the accompanying materials are made)
[//]: # (available under the terms of the Eclipse Public License 2.0)
[//]: # (which is available at https://www.eclipse.org/legal/epl-2.0/)
[//]: # (SPDX-License-Identifier: EPL-2.0)

> Please provide a summary of your changes here.
Added pagination code so that release list are loaded 10 at a time on the ECC portlet
> * Which issue is this pull request belonging to and how is it solving it? (*#1985*)

### How To Test?
Have a lot of releases (>1000) and then click on ECC portlet

### CODE REVIEW

File: frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/ecc/EccPortlet.java [hunk 1]
```diff
@@ -59,19 +87,171 @@ public void doView(RenderRequest request, RenderResponse response) throws IOExce
...
+                Collections.sort(releaseList, compareByAssessmentDate(isAsc));
+                break;
+            default:
+                break;
+        }
+
+        return releaseList;
+    }
+    
+    public static String nullToEmptyString(Object in) {
```
Comment (smrutis1) [inline frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/ecc/EccPortlet.java:219]: Please use the existing method from ` CommonUtils`

**Review by smrutis1 — CHANGES_REQUESTED** [review]
Minor change requestes

File: frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/ecc/EccPortlet.java [hunk 2]
```diff
@@ -24,6 +24,25 @@
 import java.util.Collections;
 import java.util.List;
 
+import com.liferay.portal.kernel.json.JSONArray;
```
Comment (smrutis1) [inline frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/ecc/EccPortlet.java:27]: Please re-arrange the imports

**Review by smrutis1 — CHANGES_REQUESTED** [review]

**Comment by nikkuma7** [comment]
image
Data is not loading due to timeout.

**Comment by eldrin30** [comment]
> 
Capture
There was some server and indexing issue earlier, but now its working

**Comment by nikkuma7** [comment]
Test successful.

**Review by smrutis1 — APPROVED** [review]
Code looks good.

**Pull request merged**